docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -3,189 +3,105 @@ title: Microsoft Teams
-
-```sql
-CREATE ML_ENGINE llama_index_engine
-FROM llama_index
-USING openai_api_key='your-openai-api-key';
-```
-
-```sql
-CREATE MODEL llama_index_convo_model
-PREDICT answer
-USING
-  engine = 'llama_index_engine',
-  mode = 'conversational',
-  prompt = 'answer users questions as a helpful assistant',
-  user_column = 'question',
-  assistant_column = 'answer';
-```
-
-#### Step 2.2: Create an Agent using the Conversational Model
-
-Let's create an agent using the conversational model created above:
-
-```sql
-CREATE AGENT convo_agent
-USING
-    model = 'llama_index_convo_model',
-```
-
-### Step 3: Create a [Chatbot](https://docs.mindsdb.com/agents/chatbot)
-
-Finally, create a chatbot using the agent and the Microsoft Teams data source created above:
+## Usage
 
-```sql
-CREATE CHATBOT teams_chatbot
-USING
-    database = 'teams_datasource',
-    agent = 'convo_agent';
-```
+This integration can only be used to create chatbots for Microsoft Teams via the [`CREATE CHATBOT`](/agents/chatbot) syntax. Currently, *it cannot be used as a data source for other workloads*.
+
+Follow the instructions given below to set up the Microsoft Teams app that will act as the chatbot:
+
+  1. Follow [this link](https://dev.botframework.com/bots/new) to the Microsoft Bot Framework portal and sign in with your Microsoft account.
+  2. Fill out the *Display name*, *Bot handle*, and, optionally, the *Long description*, but leave the *Messaging endpoint* field empty for now.
+  3. Set the *App type* to be 'Multi Tenant' and click on *Create Microsoft App ID and password*. This will open a new tab with the Azure portal.
+  4. Click on *New registration* and fill out the *Name* and select the 'Accounts in any organizational directory (Any Azure AD directory - Multitenant)' option under *Supported account types*, and click on *Register*. Record the *Application (client) ID* for later use.
+  5. Click on *Certificates & secrets* under *Manage*.
+  6. Click on *New client secret* and fill out the *Description* and select an appropriate *Expires* period, and click on *Add*.
+  7. Copy the client secret and save it in a secure location.
+  <Tip>
+  If you already have an existing app registration, you can use it instead of creating a new one and skip steps 4-6.
+  </Tip>
+
+  8. Open the MindsDB Editor and create a connection to Microsoft Teams using the client ID and client secret obtained in the previous steps using the SQL command provided above.
+  9. Using this connection, create a chatbot using the [`CREATE CHATBOT`](/agents/chatbot) syntax.
+  10. Run the `SHOW CHATBOTS` command and record the `webhook_token` of the chatbot you created.
+  11. Navigate back to the Microsoft Bot Framework portal and fill out the messaging endpoint in the following format: `<mindsdb_url>/api/webhooks/chatbots/<webhook_token>`.
+  <Tip>
+  The `<mindsdb_url>` is the URL of where MindsDB is running.
```

Comment:
Let's add to this sentence at the end `...running, which should be replaced with real values.`